lexicons/app/bsky/feed/getPostThread.json

```
@@ -56,6 +58,14 @@
         }
       }
     },
+    "notFoundPost": {
+      "type": "object",
+      "required": ["uri", "notFound"],
+      "properties": {
+        "uri": {"type": "string"},
+        "notFound": {"type": "boolean"}
```

Not a must-have, but wanted to toss this out as a possibility:
```suggestion
        "notFound": {"const": true}
```